rpt/hmis108_referrals.properties

```diff
@@ -6,16 +6,11 @@ Count = Count
 
 Description = Description
 
-Hmis108InpatientMonthlyReport = HMIS 108:      INPATIENT MONTHLY REPORT
+Hmis108InpatientMonthlyReport = HMIS 108: INPATIENT MONTHLY REPORT
 
 Month = Month:
 
-NoteReferredFromThisUnitReferredToThisUnit = *Note: \n- the "referred FROM \
-                                             this unit " are inpatient \
-                                             discharged the selected month.\n- \
-                                             the "referred TO this unit " are \
-                                             inpatient admitted the selected \
-                                             month.
+NoteReferredFromThisUnitReferredToThisUnit = *Note: \n- the "referred FROM this unit " are inpatient  discharged the selected month.\n- the "referred TO this unit " are inpatient admitted the selected month.
```

Comment:
should there be a space after `unit` and before the quote?  Happens twice.

Reply:
yep, thanks!